src/utils/mtev_log.c

```diff
@@ -1047,8 +1047,8 @@ jlog_logio_cleanse(mtev_log_stream_t ls) {
     if(is_datafile(entry->d_name, &logid) &&  /* make sure it is a datafile */
        logid < earliest &&                    /* and that is older enough */
        logid != log->current_log) {           /* and that isn't current */
-      char fullfile[PATH_MAX];
-      char fullidx[PATH_MAX];
+      char fullfile[PATH_MAX+PATH_MAX];
+      char fullidx[PATH_MAX+PATH_MAX];
```

Comment:
Why "PATH_MAX+PATH_MAX"?

Reply:
making sure there's enough room to snprintf later.